src/contexts/index.ts

```diff
@@ -39,12 +39,41 @@ export type SetState = (
   callback?: () => void,
 ) => void;
 
-export type DocStateContextProps = {
+export class BackBehaviorElements {
+  backBehaviorElements: Element[] = [];
```

Comment:
minor/nit: naming - since the class now confers the name spacing, that internal prop can be simplified to `elements`